addons/mail/static/src/core/common/thread.xml

```diff
@@ -48,9 +48,12 @@
             </div>
         </t>
         <t t-else="">
-            <div class="o-mail-Thread-empty d-flex flex-column align-items-center justify-content-center text-muted fst-italic h-100" t-att-class="{'p-4': props.showEmptyMessage}">
+            <div class="o-mail-Thread-empty d-flex flex-column align-items-center justify-content-center text-muted h-100" t-att-class="{'p-4': props.showEmptyMessage}">
                 <t t-if="props.thread.isLoaded and props.showEmptyMessage">
-                    <span t-ref="empty-message">There are no messages in this conversation.</span>
+                    <t name="empty-message">
+                        <span class="fs-1" style="filter: grayscale(1);">😶</span>
```

Comment:
even though it's a simple style, not sure we should do this or put it into the scss with a classname

Reply:
It's more diff and more verbose with CSS classname, and there's no perceivable performance gain with CSS, hence why I did it that way